DBM-Raids-Vanilla/VanillaSoD_Gnomeregan/MechanicalMenagerie.lua

```diff
@@ -36,7 +36,7 @@ mod:RegisterEventsInCombat(
  or (ability.id = 436828 or ability.id = 436570) and type = "cast"
 --]]
 --General
-local warnSelfRepair				= mod:NewCastAnnounce(440073, 4)
+local warnSelfRepair				= mod:NewSpecialWarningCast(440073, nil, nil, nil, 3)
```

Comment:
This is incorrect object. this is the "Stop casting" object, used for aoe interrupts usually. Misleading naming I know.
`	cast								= "%s - stop casting",`
Probably just make it NewSpecialWarningSpell
or even custom localize it since spell would just be spellname and nothing more. I don't think there actually is an auto localized object for "%s in 20 seconds" at glance over the AUTO_SPEC_WARN_TEXTS table.

Reply:
fixed; just %s from spell is what i wanted

Reply:
the "in 20 seconds" part isn't relevant, because if you let this thing go through you basically wipe. self repair means kill everything *now*